content/modules/data-table/versions/1.7.0.um

```diff
@@ -0,0 +1,26 @@
+@version 1.5.0
+  @examples
+    @example
+      @html: @inline 1.5.0/simple-example.html
```

Comment:
I guess as examples can become out of date we'll need to repeat them in later versions if we want to add more examples. Just makes it a bit more tedious